models/migrations/migrations.go

```diff
@@ -61,7 +61,6 @@ type Version struct {
 // update minDBVersion accordingly
 var migrations = []Migration{
 	// Gitea 1.5.0 ends at v69
```

Comment:
```suggestion
	// Gitea 1.5.0 ends at v69

```
an intended new line ...